### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by tashisrael**
# In this PR, I implemented the following:
- Added the species column to the animals table.
- Inserted more data into the animals table.
- Did transactions
- Wrote queries to the given questions.

# Below are the screenshots of all the results
insertdata
setUnspecified
rollback
digimon
pokemon
commit
transactionC
trasactoinD
querryQuestions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,89 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+
+/* Begin Transaction B */
+BEGIN;
+
+/* update species col for animals with names ending in mon */
+update animals set species = 'digimon' where name like '%mon';
+
+/* update species col for animals with names not ending in mon */
+update animals set species = 'pokemon' where species IS null;
+
```
Comment (AnselemOdims) [inline queries.sql:36]: - [x] Good job on this queries but please kindly include a query to confirm the update was made before committing.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @tashisrael , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by cynthiainga — COMMENTED** [review]
Hi @tashisrael, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a `README` file. I noticed that you delete some essential files like ` the README file, LICENSE file, and the logo`, please kindly restore them 🙃 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql (hunk 1 above)
Comment (tashisrael) [reply queries.sql:36]: Hello @AnselemOdims , its already confirmed as shown from the screenshots above
image

**Comment by tashisrael** [comment]
> Hi @tashisrael,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that does not have a `README` file. I noticed that you delete some essential files like ` the README file, LICENSE file, and the logo`, please kindly restore them 🙃
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by tashisrael** [comment]
Hello @cynthiainga, On my side and on GitHub, the readme file is upto date.

**Comment by cynthiainga** [comment]
> Hello @cynthiainga, On my side and on GitHub, the readme file is upto date.

Hello @tashisrael,
If you check very well you delete two files 'the README file and the LICENSE file' in the branch also you delete the logo of Microverse. You can go the `Files changed` on this Pull Request and you will see that there are files you deleted. Check very well.

**Review by Reem-lab — APPROVED** [review]
# Hi @tashisrael, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- Followed Gitflow ✅
- Great working on the functionality 🔥🚀🎉
- Descriptive PR ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**